src/tsickle_compiler_host.ts

```diff
@@ -169,7 +170,7 @@ export class TsickleCompilerHost implements ts.CompilerHost {
   }
 
   stripAndStoreExistingSourceMap(sourceFile: ts.SourceFile): ts.SourceFile {
```

Comment:
We can't, because then the return type has to be ts.SourceFile|undefined and that has bad effects higher up.  I've added a comment explaining the weirdness.